aibot/tests.py

```
@@ -1 +1,142 @@
+        self.assertEqual(resp.status_code, 500)
+        self.assertNotIn("API error", resp.json().get("error", ""))
+
+    def test_persistence_failure_does_not_affect_response(self):
+        """If CodeReview.objects.create fails, response still succeeds."""
+        mock_response = MagicMock()
+        mock_response.text = '{"summary": "ok"}'
+
+        mock_client = MagicMock()
+        mock_client.models.generate_content.return_value = mock_response
+
+        with self.settings(WORKER_SECRET="test-secret", GEMINI_API_KEY="fake-key"), \
```

Ruff (0.15.5)

[error] Possible hardcoded password assigned to argument: "WORKER_SECRET"

(S106)